k3d-modular/cert_manager.tf

```diff
@@ -0,0 +1,11 @@
+# using the cert-manager module to create a cert-manager deployment
+module "cert-manager" {
+  source = "../modules/cert-manager-module"
```

Comment:
The initial idea was to make all connections to deployed resources TLS Encrypted even with the local recipe! but apparently, I guess will probably remove it since we don't really require it in a local setting and it's failing sometimes

Reply:
okay, got it. I think it is failing because although you specify a `letsencrypt-staging` issuer in your ingress definitions, I don't see where that issuer is deployed on the cluster. 

You would need something like this to be applied to the cluster for certs to be issued.
```yaml
apiVersion: cert-manager.io/v1
kind: ClusterIssuer
metadata:
  name: letsencrypt-staging
spec:
  acme:
    server: https://acme-v02.api.letsencrypt.org/directory
    email: jayesh.ext@zenml.io
    privateKeySecretRef:
      name: letsencrypt
    solvers:
    - http01:
        ingress:
          class: nginx
```
 
 Or, you can also remove the cert-manager altogether.